spec/features/brexit_checker/accounts_spec.rb

```diff
@@ -49,7 +49,7 @@
       @original_headers = page.driver.options[:headers]
       @original_account_session_header = "placeholder-session"
       page.driver.options[:headers] ||= {}
-      page.driver.options[:headers].merge!("GOVUK-Account-Session" => @original_account_session_header)
+      page.driver.options[:headers].merge!("HTTP_GOVUK_ACCOUNT_SESSION" => @original_account_session_header)
```

Comment:
No, the request header sent to Rails is `GOVUK-Account-Session`.  The name-munging happens inside the Rails middleware.

See the frontend tests: https://github.com/alphagov/frontend/blob/9ce9aaf8248715c0fac8df87a293232c4e48b4da/test/support/session_helpers.rb#L3 https://github.com/alphagov/frontend/blob/master/test/integration/sessions_test.rb#L9

Reply:
Discussed offline: this actually had to be `HTTP_GOVUK_ACCOUNT_SESSION` because `page.driver.options[:headers]` doesn't mangle the header names... but we should have been using `page.driver.header` anyway.